exonum/src/events/network.rs

```diff
@@ -76,6 +76,21 @@ pub enum NetworkRequest {
     Shutdown,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Copy, PartialEq)]
+pub struct HttpBackendConfig {
+    pub restart_retry_attempts: u64,
```

Comment:
Hm. IMO `u16` is enough in both cases.

Reply:
It would be good to add a comment doc for a public field.